libmozdata/config.py

```diff
@@ -63,6 +63,9 @@ def set_config(conf):
     __config = conf
 
 
-def get(section, option, default=None, type=str):
+def get(section, option, default=None, type=str, required=False):
     global __config
-    return __config.get(section, option, default=default, type=type)
+    value = __config.get(section, option, default=default, type=type)
+    if required:
+        assert value, f"Option {option} in section {section} is not set"
```

Comment:
```suggestion
        assert value is not None, f"Option {option} in section {section} is not set"
```

Otherwise it would fail with a config value of `0` or `False`